media/css/products/relay/components/reviews.scss

```
@@ -57,10 +57,14 @@ $brand-theme: 'firefox';
 }
 
 .c-review-amo-title {
-    @include text-display-sm;
     font-weight: bold;
+    line-height: 1;
     padding-bottom: $spacing-lg;
     text-transform: uppercase;
+
+    html[lang^='en'] & {
+        @include text-display-sm;
```

Sorry I think I gave the wrong mixin name on my last review, not sure what I was thinking 🤦  This should be `text-title-sm`

```suggestion
        @include text-title-sm;
```